neptune/new/internal/backends/hosted_file_operations.py

```diff
@@ -289,8 +289,12 @@ def _build_multipart_urlset(
 ) -> MultipartUrlSet:
     urlnameset = MULTIPART_URLS[target]
     return MultipartUrlSet(
-        start_chunked=getattr(swagger_client.api, urlnameset.start_chunked),
-        finish_chunked=getattr(swagger_client.api, urlnameset.finish_chunked),
+        start_chunked=with_api_exceptions_handler(
+            getattr(swagger_client.api, urlnameset.start_chunked)
+        ),
+        finish_chunked=with_api_exceptions_handler(
+            getattr(swagger_client.api, urlnameset.finish_chunked)
+        ),
```

Comment:
Is chunk upload call retried?

Reply:
yes, it uses internally `upload_raw_data` function, which already had a `@with_api_exceptions_handler` annotation